src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/Config/NewAzureRmVmssConfigCommand.cs

```diff
@@ -149,6 +149,14 @@ public partial class NewAzureRmVmssConfigCommand : Microsoft.Azure.Commands.Reso
             Mandatory = false)]
         public SwitchParameter AutoOSUpgrade { get; set; }
 
+        [Parameter(
+            Mandatory = false)]
+        public SwitchParameter EnableAutoRollback { get; set; }
```

Comment:
@hyonholee do we want to allow users to provide `EnableAutoRollback` and `DisableAutoRollback` in the same parameter set?